docs/references/configuration.rst

```diff
@@ -596,14 +596,14 @@ jwt-aud
 -------
 
   =============== =================================
-  **Type**        String
+  **Type**        String (must be a valid regular expression)
   **Default**     `n/a`
```

Comment:
```suggestion
  **Default**     .*
```

Perhaps also mention in the description something like: "For example, the default value `.*` matches any `aud` claim."?